test/Microsoft.AspNetCore.JsonPatch.Test/DictionaryAdapterTest.cs

```diff
@@ -3,7 +3,6 @@
 
 using System;
 using System.Collections.Generic;
```

Comment:
Do the expando object tests need an update?

Reply:
There are no unit tests for ExpandoObjectAdapter. Only integration tests, which pass using the new adapter

Reply:
lel ok